dm/syncer/data_validator.go

```diff
@@ -628,15 +665,16 @@ func (v *DataValidator) doValidate() {
 		default:
 			currLoc.Position.Pos = e.Header.LogPos
 		}
-
+		// update validator metric
+		v.updateValidatorBinlogMetric(currLoc)
```

Comment:
maybe we can make it near `v.updateValidatorBinlogLag(currLoc)`, move this part down or reversely